### PR TITLE
fix-vtex-connector-v2-pix_pt

### PR DESCRIPTION
**Pull request by douglas-cruz**
Comunicar en el Devsite la nueva versión del gateway MercadoPagoV2 en VTEX para Brasil, que incluye Pix como medio de pago. 

## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/plugins/unofficial/vtex.pt.md [hunk 1]
```diff
@@ -52,9 +57,49 @@ Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga
 
 E pronto! Sua afiliação com o Mercado Pago já está ativa!
 
+### Criando afiliação de gateway MercadoPagoV2 com Pix
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:60]: @abderossi ¿no dijimos de sacarlo al "com Pix"?

File: guides/plugins/unofficial/vtex.pt.md [hunk 2]
```diff
@@ -52,9 +57,49 @@ Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga
...
+
+### Requisito prévio
+#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Obtenha uma chave Pix
+
+Para começar, você deve ter cadastrada uma chave Pix na conta do vendedor. Este dado é único, serve para identificar sua conta e permitirá que você utilize as funcionalidades do meio de pagamento.
+
+[Conheça como criar uma chave Pix](https://www.mercadopago.com.br/ajuda/17843)
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:67]:
```suggestion
[Conheça como criar uma chave Pix](https://www.mercadopago.com.br/stop/pix?url=https%3A%2F%2Fwww.mercadopago.com.br%2Fadmin-pix-keys%2Fmy-keys&authentication_mode=required)
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 3]
```diff
@@ -143,13 +202,6 @@ Para criar essa condição de pagamento, siga os passos abaixo:
...
-<!-- -->
-> NOTE
-> 
-> Nota
->
-> As mudanças nas Condições de pagamento podem levar até 10 minutos para serem aplicadas. <br>
-
 ## Configurando o parcelamento em sua conta Mercado Pago
 
 1. Acesse sua [conta Mercado Pago](https://www.mercadopago.com.br/business/) e clique em "Seu negócio"
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:207]:
```suggestion
1. Acesse sua [conta Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]/business) e clique em "Seu negócio"
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 VTEX é uma **plataforma e-commerce para sua loja virtual**, que permite processar pagamentos através do Mercado Pago.
 
-Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito) e [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) no próprio [checkout da sua loja ou redirecionado para o site do Mercado Pago](#bookmark_condição_de_pagamento_checkout_pro).
+Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) e [Pix](#bookmark_condição_de_pagamento_Pix) no próprio checkout da sua loja ou [redirecionado para o site do Mercado Pago](#bookmark_condição_de_pagamento_checkout_pro).
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:7]:
```suggestion
Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) e [Pix](#bookmark_condição_de_pagamento_pix) no próprio checkout da sua loja ou [redirecionado para o site do Mercado Pago](#bookmark_condição_de_pagamento_checkout_pro).
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 5]
```diff
@@ -16,11 +16,16 @@ Os **passos para começar a operar com Mercado Pago**, são os seguintes:
...
+Uma afiliação de gateway é um conjunto de configurações que representam o processamento de seus pagamentos com o Mercado Pago.
+Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) ou [Checkout Pro](#bookmark_condição_de_pagamento_checkout_pro), e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_Pix).
+
+### Criando afiliação de gateway MercadoPagoV1
 
 Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga os passos abaixo:
 
-1. No painel de administração de sua loja, acesse Configurações do módulo de pagamentos.
+1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.
 2. Na aba Afiliações de Gateways, clique no botão "+".
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:29]:
```suggestion
2. Na aba "Afiliações de Gateways", clique no botão "+".
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 6]
```diff
@@ -52,9 +57,49 @@ Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga
...
+Para começar, você deve ter cadastrada uma chave Pix na conta do vendedor. Este dado é único, serve para identificar sua conta e permitirá que você utilize as funcionalidades do meio de pagamento.
+
+[Conheça como criar uma chave Pix](https://www.mercadopago.com.br/ajuda/17843)
+
+> Considere que o registro da chave Pix pode levar alguns minutos.
+
+Para criar uma **afiliação de gateway de pagamento Pix com o MercadoPagoV2**, siga os passos abaixo:
+
+1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.
+2. Na aba Afiliações de Gateways, clique no botão "+".
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:74]:
```suggestion
2. Na aba "Afiliações de Gateways", clique no botão "+".
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 7]
```diff
@@ -52,9 +57,49 @@ Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga
 
 E pronto! Sua afiliação com o Mercado Pago já está ativa!
 
+### Criando afiliação de gateway MercadoPagoV2 com Pix
+
+### Requisito prévio
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:62]:
```suggestion
#### Requisito prévio
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 8]
```diff
@@ -143,13 +202,6 @@ Para criar essa condição de pagamento, siga os passos abaixo:
 >
 > Caso as condições de pagamento MercadoPagoPro e MercadoPagoWallet não estiverem disponíveis para adicioná-las, certifique em sua loja de aplicativos da VTEX que o App Mercado Pago Payment APP esteja instalado ou solicite a instalação pela equipe VTEX sob demanda através de ticket no [Support VTEX](https://support.vtex.com/). <br>
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:203]:
```suggestion
> Caso as condições de pagamento MercadoPagoPro e MercadoPagoWallet não estiverem disponíveis para adicioná-las, certifique em sua loja de aplicativos da VTEX que o App Mercado Pago Payment APP esteja instalado ou solicite a instalação pela equipe VTEX sob demanda através de ticket no [Support VTEX](https://support.vtex.com). <br>
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 9]
```diff
@@ -52,9 +57,49 @@ Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga
...
+
+![Setting affiliation in VTEX](/images/vtex/vtex_afiliacao_gateway_v2.gif)
+
+E pronto! Sua afiliação com o Mercado Pago já está ativa!
+
+> WARNING
+>
+> Importante
+>
+> Lembre-se que o gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix) por enquanto adiciona apenas o Pix como meio de pagamento. Para configurar outros métodos de pagamento, crie uma afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1).
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:91]:
```suggestion
> Lembre-se que o gateway MercadoPagoV2 por enquanto adiciona apenas o Pix como meio de pagamento. Para configurar outros métodos de pagamento, crie uma afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1).
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 10]
```diff
@@ -73,21 +118,35 @@ Para criar uma **condição de pagamento cartão de crédito** utilizando sua af
...
 
+### Condição de pagamento Pix
+
+Para **criar uma condição de pagamento Pix utilizando sua afiliação com o [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, siga os passos abaixo:
+
+1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.
+2. Na aba "Condições de pagamento", clique no botão "+".
+3. Na seção "Boleto", escolha "Boleto Bancário" para adicionar ao meio de pagamento.
+4. Informe o "nome da regra" para facilitar a identificação e ative a condição de pagamento no campo "Status".
+5. Informe sua afiliação com o **[MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, no campo "Processar com a afiliação".
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:141]:
```suggestion
5. Informe sua afiliação com o MercadoPagoV2, no campo "Processar com a afiliação".
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 11]
```diff
@@ -73,21 +118,35 @@ Para criar uma **condição de pagamento cartão de crédito** utilizando sua af
...
 
 ![Setting payments in VTEX](/images/vtex/vtex_condicao_pagamento_boleto_3.gif)
 
+### Condição de pagamento Pix
+
+Para **criar uma condição de pagamento Pix utilizando sua afiliação com o [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, siga os passos abaixo:
+
+1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.
+2. Na aba "Condições de pagamento", clique no botão "+".
+3. Na seção "Boleto", escolha "Boleto Bancário" para adicionar ao meio de pagamento.
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:139]:
```suggestion
3. Na seção “Pagamento Instantâneo”, escolha o meio de pagamento Pix.
```

File: guides/resources/changelog/current-year.en.md [hunk 12]
```diff
@@ -7,6 +7,22 @@ Find out everything about the new versions and updates of Mercado Pago integrati
...
+>
+> New version of MercadoPagoV2 gateway for VTEX
+>
+> NEWS: FEATURE
+>
+> Product: VTEX
+
+Now you can include Pix as a payment method with the gateway MercadoPagoV2 for VTEX.
+
+Go to [VTEX](https://www.mercadopago.com.br/developers/pt/guides/plugins/unofficial/vtex) documentation
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:23]:
```suggestion
[Go to VTEX documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/plugins/unofficial/vtex)
```

File: guides/resources/changelog/current-year.es.md [hunk 13]
```diff
@@ -7,6 +7,22 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
...
+>
+> Nueva versión de gateway MercadoPagoV2 para VTEX
+>
+> NEWS: FEATURE
+>
+> Product: VTEX
+
+Ahora puedes incluir Pix como medio de pago con el gateway MercadoPagoV2 para VTEX.
+
+Ir a la documentación de [VTEX](https://www.mercadopago.com.br/developers/pt/guides/plugins/unofficial/vtex).
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:23]:
```suggestion
[Ir a la documentación de VTEX](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/plugins/unofficial/vtex)
```

File: guides/resources/changelog/current-year.pt.md [hunk 14]
```diff
@@ -6,6 +6,23 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
...
+>
+> Nova versão do gateway MercadoPagoV2 para VTEX
+>
+> NEWS: FEATURE
+>
+> Product: VTEX
+
+Agora você pode incluir Pix como meio de pagamento com o gateway MercadoPagoV2 para VTEX.
+
+Ir para a documentação [VTEX](https://www.mercadopago.com.br/developers/pt/guides/plugins/unofficial/vtex).
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:23]:
```suggestion
[Ir para a documentação VTEX](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/plugins/unofficial/vtex)
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 15]
```diff
@@ -73,21 +118,35 @@ Para criar uma **condição de pagamento cartão de crédito** utilizando sua af
...
+5. Informe sua afiliação com o **[MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, no campo "Processar com a afiliação".
+6. Clique em "Salvar".
+<p>&nbsp;</p>
+
+![Setting payments in VTEX](/images/vtex/vtex_condicao_pagamento_pix.gif)
+
 ### Condição de pagamento Checkout Pro
 
-O Checkout Pro é a integração que te permite cobrar através do nosso formulário web de forma simples, rápida e segura.
+O [Checkout Pro](https://www.mercadopago.com.br/developers/pt/guides/online-payments/checkout-pro/introduction) é a integração que te permite cobrar através do Mercado Pago, contando com as meios de pagamentos como Pix, boleto parcelado, débito virtual da Caixa e carteira digital do Mercado Pago.
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:149]:
```suggestion
O [Checkout Pro](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-pro/introduction) é a integração que te permite cobrar através do Mercado Pago, contando com as meios de pagamentos como Pix, boleto parcelado, débito virtual da Caixa e carteira digital do Mercado Pago.
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by abderossi — CHANGES_REQUESTED** [review]

File: guides/plugins/unofficial/vtex.pt.md [hunk 16]
```diff
@@ -16,12 +16,17 @@ Os **passos para começar a operar com Mercado Pago**, são os seguintes:
 3. Configure as condições de pagamento Mercado Pago.
 4. Configure device fingerprint.
 
-## Criando afiliação de gateway Mercado Pago
+## Cadastre uma afiliações de gateway
+
+Uma afiliação de gateway é um conjunto de configurações que representam o processamento de seus pagamentos com o Mercado Pago.
+Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) ou [Checkout Pro](#bookmark_condição_de_pagamento_checkout_pro), e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_Pix).
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:22]:
```suggestion
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_pix).
```

Comment (abderossi) [inline guides/plugins/unofficial/vtex.pt.md]:
```suggestion
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2)** para o processamento de pagamento Pix.
```

**Review by abderossi — CHANGES_REQUESTED** [review]

File: guides/plugins/unofficial/vtex.pt.md [hunk 17]
```diff
@@ -16,12 +16,17 @@ Os **passos para começar a operar com Mercado Pago**, são os seguintes:
 3. Configure as condições de pagamento Mercado Pago.
 4. Configure device fingerprint.
 
-## Criando afiliação de gateway Mercado Pago
+## Cadastre uma afiliações de gateway
+
+Uma afiliação de gateway é um conjunto de configurações que representam o processamento de seus pagamentos com o Mercado Pago.
+Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_pix).
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:22]:
```suggestion
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_pix).
```

File: guides/plugins/unofficial/vtex.pt.md [hunk 18]
```diff
@@ -73,21 +118,35 @@ Para criar uma **condição de pagamento cartão de crédito** utilizando sua af
...
-5. Informe sua afiliação com o Mercado Pago, no campo "Processar com a afiliação".
+5. Informe sua afiliação com o MercadoPagoV1, no campo "Processar com a afiliação".
 6. Clique em "Salvar".
 <p>&nbsp;</p>
 
 ![Setting payments in VTEX](/images/vtex/vtex_condicao_pagamento_boleto_3.gif)
 
+### Condição de pagamento Pix
+
+Para **criar uma condição de pagamento Pix utilizando sua afiliação com o [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, siga os passos abaixo:
```
Comment (pamalv7) [inline guides/plugins/unofficial/vtex.pt.md:135]:
```suggestion
Para **criar uma condição de pagamento Pix utilizando sua afiliação com o [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2)**, siga os passos abaixo:
```

**Review by abderossi — APPROVED** [review]

**Review by pamalv7 — APPROVED** [review]

**Review by abderossi — APPROVED** [review]

**Pull request merged**